### PR TITLE
Remove PressedCollection updatedBy and updatedEmail

### PR DESCRIPTION
**Pull request by nicl**
These fields introduce unused PI data; let's remove them.

Should have no impact on users.

### CODE REVIEW

File: common/app/model/facia/PressedCollection.scala [hunk 1]
```diff
@@ -17,8 +17,8 @@ case class PressedCollection(
   backfill: List[PressedContent],
   treats: List[PressedContent],
   lastUpdated: Option[DateTime],
-  updatedBy: Option[String],
-  updatedEmail: Option[String],
+//  updatedBy: Option[String],
+//  updatedEmail: Option[String],
```
Comment (TBonnin) [inline common/app/model/facia/PressedCollection.scala]: remove them all together rather than commenting them

**Review by TBonnin — CHANGES_REQUESTED** [review]
Does that require repressing?

File: common/app/model/facia/PressedCollection.scala (hunk 1 above)
Comment (nicl) [reply common/app/model/facia/PressedCollection.scala]: ah thanks.

**Comment by nicl** [comment]
@TBonnin re requiring re-pressing, I'd assumed not as the fields are optional, but I haven't tested on CODE yet.

**Review by TBonnin — APPROVED** [review]

**Pull request merged**